src/DynamoCoreWpf/UI/Converters.cs

```diff
@@ -1754,7 +1754,7 @@ public object Convert(object value, Type targetType, object parameter, CultureIn
                 return input;
```

Comment:
I think "none" string needs to be localized, please put it into `Resources.resx` file for `DynamoCoreWpf` project. Once that's done, remember to copy the `Resources.resx` file to overwrite `Resources.en-US.resx` so that they are completely identical.